js/ui/range_slider.js

```diff
@@ -297,6 +307,20 @@ const RangeSlider = Slider.inherit({
         this.option('value', [start, end]);
     },
 
+    _getActualStartValue: function() {
+        if(!this._actualValue) {
+            return undefined;
+        }
+        return this._actualValue[0];
+    },
+
+    _getActualEndValue: function() {
```

Comment:
I think it's a good idea to implement only one function `_getActualValue` instead `_getActualStartValue` and `_getActualEndValue` to reduce copypaste and simplify the code.